packages/react/src/patterns/sub-patterns/TableOfContents/README.md

```diff
@@ -83,11 +83,26 @@ Add the following line in your `.env` file at the root of your project.
 
 | Name           | Required | Data Type | Default Value | Description                                                                      |
 | -------------- | -------- | --------- | ------------- | -------------------------------------------------------------------------------- |
-| `children`     | NO       | Array     | null          | Content to display next to the side nav.                                         |
+| `children`     | NO       | Array     | null          | Content to display next to the side nav. (See `Children` bellow)                 |
```

Comment:
I wonder if we should call this something else other than `children`. Children is also a reserved name for the actual children content that goes into the content area of the Table of Contents. 

Maybe we should rename to something like `headingContent`?